.github/workflows/promote-to-latest.yml

```diff
@@ -52,35 +53,30 @@ jobs:
           exit 1
         fi
         set +x
-        # x64
-        docker pull nextcloud/$AIO_NAME\:beta
-        docker tag nextcloud/$AIO_NAME\:beta nextcloud/$AIO_NAME\:latest
-        docker push nextcloud/$AIO_NAME\:latest
-        docker tag nextcloud/$AIO_NAME\:beta nextcloud/$AIO_NAME\:"$DATE"-latest
-        docker push nextcloud/$AIO_NAME\:"$DATE"-latest
-        if [ "$AIO_NAME" = "aio-nextcloud" ]; then
-          set -x
-          PHP_VERSION="$(docker inspect nextcloud/aio-nextcloud:beta | grep PHP_VERSION | grep -oP '[0-8.]+' | sed 's|\.[0-9]\+$||')"
-          if [ -n "$PHP_VERSION" ]; then
-            docker tag nextcloud/aio-nextcloud\:beta nextcloud/aio-nextcloud\:php"$PHP_VERSION"-latest
-            docker push nextcloud/aio-nextcloud\:php"$PHP_VERSION"-latest
-          fi
-          set +x
-        fi
+        
+        # multiarch
+        docker buildx imagetools create --tag nextcloud/$AIO_NAME\:latest nextcloud/$AIO_NAME\:beta
+        docker buildx imagetools create --tag nextcloud/$AIO_NAME\:"$DATE"-latest nextcloud/$AIO_NAME\:beta
+        
+        # arm64
         if [ "$AIO_NAME" != "aio-clamav" ]; then
-          # arm64 
-          docker pull nextcloud/$AIO_NAME\:beta-arm64
-          docker tag nextcloud/$AIO_NAME\:beta-arm64 nextcloud/$AIO_NAME\:latest-arm64
-          docker push nextcloud/$AIO_NAME\:latest-arm64
-          docker tag nextcloud/$AIO_NAME\:beta-arm64 nextcloud/$AIO_NAME\:"$DATE"-latest-arm64
-          docker push nextcloud/$AIO_NAME\:"$DATE"-latest-arm64
-          if [ "$AIO_NAME" = "aio-nextcloud" ]; then
-          set -x
-          PHP_VERSION="$(docker inspect nextcloud/aio-nextcloud:beta-arm64 | grep PHP_VERSION | grep -oP '[0-8.]+' | sed 's|\.[0-9]\+$||')"
-          if [ -n "$PHP_VERSION" ]; then
-            docker tag nextcloud/aio-nextcloud\:beta-arm64 nextcloud/aio-nextcloud\:php"$PHP_VERSION"-latest-arm64
-            docker push nextcloud/aio-nextcloud\:php"$PHP_VERSION"-latest-arm64
-          fi
-          set +x
+        docker buildx imagetools create --tag nextcloud/$AIO_NAME\:latest-arm64 nextcloud/$AIO_NAME\:beta-arm64
+        docker buildx imagetools create --tag nextcloud/$AIO_NAME\:"$DATE"-latest-arm64 nextcloud/$AIO_NAME\:beta-arm64
         fi
+        
+        if [ "$AIO_NAME" == "aio-nextcloud" ]; then
+            set -x
+            docker pull nextcloud/$AIO_NAME\:beta
+            PHP_VERSION="$(docker inspect nextcloud/$AIO_NAME\:beta | grep PHP_VERSION | grep -oP '[0-8.]+' | sed 's|\.[0-9]\+$||')"
+            docker image remove nextcloud/$AIO_NAME\:beta
```

Comment:
```suggestion
            docker image remove --force nextcloud/$AIO_NAME\:beta
```

Reply:
Not sure how important it is to remove the image locally again?

Reply:
I think the local manifest pulled is only single arch and I dont know if it is true, but that could cause that in the following build steps only a single arch image is copied

Reply:
okay, fine by me to remove that then 👍